### PR TITLE
 Add tip how to reset the db to setup guide - Closes #1099

### LINKED ISSUE #1099: Add note how to reset the application database
The docs are missing a description how to reset the application data, including the database data.

The location of the application files should be explained, and a code snippet how to remove the files should be presented.

### PR DESCRIPTION
**Pull request by Tschakki**
Closes #1099

### CODE REVIEW

File: modules/ROOT/pages/guides/app-development/setup.adoc [hunk 1]
```diff
@@ -325,6 +325,27 @@ EXAMPLES
...
+Once the application is started for the first time, it will save the application-specific data under the path `~/.lisk/new_app`.
+
+To reset the database of the application, simply delete the folder with the application data:
+
+[source,bash]
+----
+rm -r ~/.lisk/new_app/
+----
+
+To remove the data of all application, execute:
```
Comment (chris529) [inline modules/ROOT/pages/guides/app-development/setup.adoc]:
```suggestion
To remove all data from the application, execute the following command:
```

File: modules/ROOT/pages/guides/app-development/setup.adoc [hunk 2]
```diff
@@ -325,6 +325,27 @@ EXAMPLES
...
+
+[source,bash]
+----
+rm -r ~/.lisk/
+----
+====
+
 === Next steps
 
 By installing Lisk Commander and running `lisk init`, you have now a working blockchain application with the default configurations for runnning in a local devnet.
```
Comment (chris529) [inline modules/ROOT/pages/guides/app-development/setup.adoc]:
```suggestion
By installing Lisk Commander and running `lisk init`, a working blockchain application now exists with the default configurations for running in a local devnet.
```

**Review by chris529 — COMMENTED** [review]
Looks good, just 2 minor points 👍

File: modules/ROOT/pages/guides/app-development/setup.adoc (hunk 1 above)
Comment (Tschakki) [reply modules/ROOT/pages/guides/app-development/setup.adoc]:
```suggestion
To remove the data of all applications, execute:
```

File: modules/ROOT/pages/guides/app-development/setup.adoc [hunk 3]
```diff
@@ -325,9 +325,30 @@ EXAMPLES
...
+Once the application is started for the first time, it will save the application-specific data under the path `~/.lisk/new_app`.
+
+To reset the database of the application, simply delete the folder with the application data:
+
+[source,bash]
+----
+rm -r ~/.lisk/new_app/
+----
+
+To remove the data of all applications, execute:
```
Comment (chris529) [inline modules/ROOT/pages/guides/app-development/setup.adoc]:
```suggestion
To remove the data from all applications, execute:
```

**Review by chris529 — COMMENTED** [review]
as discussed 👍

**Review by chris529 — APPROVED** [review]
LGTM!

**Pull request merged**